packages/vite/src/node/ssr/ssrModuleLoader.ts

```diff
@@ -181,6 +181,18 @@ async function instantiateModule(
     }
   }
 
+  let sourceMapSuffix = ''
+  if (result.map) {
+    const moduleSourceMap = Object.assign({}, result.map, {
+      // offset the first three lines of the module (function declaration and 'use strict')
+      mappings: ';;;' + result.map.mappings,
+    })
+    // The ${'//'} is to avoid being replaced by node/server/transformRequest
```

Comment:
When does this replace happen? I tried to reproduce this one but failed to.

Reply:
It was happening in tests

Reply:
`pnpm run test` doesn't seem to fail. But I guess it happens when vite is transformed by vitest because of https://github.com/thlorenz/convert-source-map/issues/63?

Would you add a link to https://github.com/thlorenz/convert-source-map/issues/63 in the comment to make it clear?